hack/e2e.sh

```diff
@@ -7,10 +7,25 @@
 
 # Prerequirements:
 #   * pull secret (with registry.svc.ci.openshift.org) part in `/tmp/cluster/pull-secret`
+#   * CINCINNATI_IMAGE (optional) - image with graph-builder and policy-engine
 #   * env var IMAGE_FORMAT (e.g `registry.svc.ci.openshift.org/ci-op-ish8m5dt/stable:${component}`)
```

Comment:
Are we using this anywhere? If not I suggest removing it now which will make the script a bit simpler.

Reply:
`IMAGE_FORMAT` is passed by our CI, it is used to compose 
a pullspec to the deployment image. In dev builds users would override that with `CINCINNATI_IMAGE` - these sadly cannot be reconciled as the var names are hardcoded in cluster-launch templates on CI